spec/vcr/services/controlled_terms/genre_canonicable.yml

```diff
@@ -0,0 +1,121 @@
+        `call''"},{"id":13,"trace":"actionpack (5.2.4.2) lib/action_dispatch/middleware/static.rb:127:in
+        `call''"},{"id":14,"trace":"rack (2.2.2) lib/rack/sendfile.rb:110:in `call''"},{"id":15,"trace":"railties
+        (5.2.4.2) lib/rails/engine.rb:524:in `call''"},{"id":16,"trace":"puma (4.3.3)
+        lib/puma/configuration.rb:228:in `call''"},{"id":17,"trace":"puma (4.3.3)
+        lib/puma/server.rb:682:in `handle_request''"},{"id":18,"trace":"puma (4.3.3)
+        lib/puma/server.rb:472:in `process_client''"},{"id":19,"trace":"puma (4.3.3)
+        lib/puma/server.rb:328:in `block in run''"},{"id":20,"trace":"puma (4.3.3)
+        lib/puma/thread_pool.rb:134:in `block in spawn_thread''"}],"Full Trace":[{"id":0,"trace":"actionpack
+        (5.2.4.2) lib/action_dispatch/middleware/debug_exceptions.rb:65:in `call''"},{"id":1,"trace":"actionpack
+        (5.2.4.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call''"},{"id":2,"trace":"railties
+        (5.2.4.2) lib/rails/rack/logger.rb:38:in `call_app''"},{"id":3,"trace":"railties
+        (5.2.4.2) lib/rails/rack/logger.rb:26:in `block in call''"},{"id":4,"trace":"activesupport
+        (5.2.4.2) lib/active_support/tagged_logging.rb:71:in `block in tagged''"},{"id":5,"trace":"activesupport
+        (5.2.4.2) lib/active_support/tagged_logging.rb:28:in `tagged''"},{"id":6,"trace":"activesupport
+        (5.2.4.2) lib/active_support/tagged_logging.rb:71:in `tagged''"},{"id":7,"trace":"railties
+        (5.2.4.2) lib/rails/rack/logger.rb:26:in `call''"},{"id":8,"trace":"actionpack
+        (5.2.4.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call''"},{"id":9,"trace":"actionpack
+        (5.2.4.2) lib/action_dispatch/middleware/request_id.rb:27:in `call''"},{"id":10,"trace":"rack
+        (2.2.2) lib/rack/runtime.rb:22:in `call''"},{"id":11,"trace":"activesupport
+        (5.2.4.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in
+        `call''"},{"id":12,"trace":"actionpack (5.2.4.2) lib/action_dispatch/middleware/executor.rb:14:in
+        `call''"},{"id":13,"trace":"actionpack (5.2.4.2) lib/action_dispatch/middleware/static.rb:127:in
+        `call''"},{"id":14,"trace":"rack (2.2.2) lib/rack/sendfile.rb:110:in `call''"},{"id":15,"trace":"railties
+        (5.2.4.2) lib/rails/engine.rb:524:in `call''"},{"id":16,"trace":"puma (4.3.3)
+        lib/puma/configuration.rb:228:in `call''"},{"id":17,"trace":"puma (4.3.3)
+        lib/puma/server.rb:682:in `handle_request''"},{"id":18,"trace":"puma (4.3.3)
+        lib/puma/server.rb:472:in `process_client''"},{"id":19,"trace":"puma (4.3.3)
+        lib/puma/server.rb:328:in `block in run''"},{"id":20,"trace":"puma (4.3.3)
+        lib/puma/thread_pool.rb:134:in `block in spawn_thread''"}]}}'
+  recorded_at: Thu, 03 Sep 2020 18:56:02 GMT
+- request:
+    method: get
+    uri: http://127.0.0.1:3001/qa//search/loc/graphicMaterials?q=tgm008084
```

Comment:
Noticing a lot of `qa//` in the URLs of Auth API calls in these VCR files. Not necessarily a big issue, but maybe check and make sure the service is generating URLs correctly?